File: C++/2021/S2.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int m, n, k;cin >> m;cin >> n;cin >> k;
    vector<int> rows(m, 0); vector<int> columns(n, 0);

    for (int q = 0; q < k; q++) {
      char rc;cin >> rc;int index;cin >> index;index--;
      if (rc == 'R') {
        rows[index] = ~rows[index];
      }
      else {
        columns[index] = ~columns[index];
      }
    }

    int output = 0;
    for (int q = 0; q < n; q++) for (int w = 0; w < m; w++) {
      output += columns[q] ^ rows[w];
    }
    cout << abs(output);
}

File: Python/2021/S2.py
# PASSED

r, c = int(input()), int(input())

rows = [0] * r
columns = [0] * c

for _ in range(int(input())):
    rc, i = input().split()

    if rc == 'R':
        rows[int(i) - 1] += 1
    else:
        columns[int(i) - 1] += 1

n = 0
for q in rows:
    for w in columns:
        n += 1 if (w + q) % 2 else 0

print(n)